.github/workflows/ci.yml

```diff
@@ -6,13 +6,7 @@ on:
       - master
 jobs:
   build-and-test:
-    runs-on: ${{ matrix.os }}
-    env:
-      USE_BAZEL_VERSION: ${{ matrix.bazel_version }}
-    strategy:
-      matrix:
-        os: [ubuntu-24.04]
-        bazel_version: [bazelbuild/7.3.1]
```

Comment:
Does removing this mean we don't benefit from the `setup-bazel` action below?

Reply:
I think we still need that so we can get Bazelisk, but correct me if I'm wrong.